### DISCUSSION TITLE
ARM64 Sonoma is missing gcc/current symlink

### DISCUSSION THREAD

**Discussion by PhilipDeegan** [Homebrew/discussions#5461] [Everyday usage]
### Output of `brew config`

```shell
I don't have access to this
```

### Output of `brew doctor`

```shell
I don't have access to this
```

### Description of issue

ARM64 Sonoma is missing gcc/current symlink

We're seeing this on `macos-latest` github actions, macos 12 and 13 have the symlink

https://github.com/orgs/community/discussions/124141

**Comment by PhilipDeegan** [comment]
For some reason [the symlink](https://github.com/Homebrew/homebrew-core/blob/master/Formula/g/gcc.rb#L151) is not installing

**Comment by Bo98** [comment]
Works fine for me locally:

```
$ ls "$(brew --prefix gcc)/lib/gcc/current"
gcc                      libgomp.a                libquadmath.dylib
libatomic.1.dylib        libgomp.dylib            libssp.0.dylib
libatomic.a              libgomp.spec             libssp.a
libatomic.dylib          libitm.1.dylib           libssp.dylib
libcc1.0.so              libitm.a                 libssp_nonshared.a
libcc1.so                libitm.dylib             libstdc++.6.dylib
libgcc_s.1.1.dylib       libitm.spec              libstdc++.6.dylib-gdb.py
libgfortran.5.dylib      libobjc-gnu.4.dylib      libstdc++.a
libgfortran.a            libobjc-gnu.a            libstdc++.dylib
libgfortran.dylib        libobjc-gnu.dylib        libstdc++exp.a
libgfortran.spec         libquadmath.0.dylib      libstdc++fs.a
libgomp.1.dylib          libquadmath.a            libsupc++.a
```

**Reply by Bo98** [reply]
The issue you may be seeing is because GCC 14 isn't actually available on GitHub Actions yet, but it will be next week: https://github.com/actions/runner-images/issues/10026

**Reply by PhilipDeegan** [reply]
thanks, it's starting to make more sense now

**Comment by gromgit** [comment]
Taking the discussion you posted literally, you're almost certainly looking in the wrong place. Homebrew ARM64 installs everything under `/opt/homebrew`, not `/usr/local`. You should always use `$HOMEBREW_PREFIX` as the base path (or `$(brew --prefix)` if it's not set).

**Reply by PhilipDeegan** [reply]
I'm not a mac person, I thought gcc was just from homebrew, but apparently not, so that makes sense
